src/general/cpuInfo.go

```diff
@@ -47,29 +47,51 @@ func NewCPULoad() *CPULoad {
 	return &CPULoad{}
 }
 
+// ReadCPULoad reads /proc/stat and returns the average load
+func ReadCPULoad() ([10]float64, error) {
+	data, error := os.ReadFile("/proc/stat")
+	if error != nil {
+		return [10]float64{}, error
+	}
+	string_data := string(data)
+	lines := strings.Split(string_data, "\n")
+	vals := strings.Split(lines[0], " ")
+	var avg [10]float64
+	sum := 0
+	for i, x := range vals {
+		if i < 2 {
+			continue
+		}
```

Comment:
Why start from index 2? Isn't the line of the format `cpu <user> <nice> <system> ... `?

Reply:
Ah yes. It kept crashing, then I realised there's 2 spaces and not one between CPU and <user>